### PR TITLE
SAAS-413 Return check description

### PR DESCRIPTION
**Pull request by palash25**
[SAAS-413](https://jira.percona.com/browse/SAAS-413)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1200

- [x] Links to other linked pull requests (optional):  https://github.com/percona/pmm/pull/601

---
- [x] Tests passed.
- [x] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: Gopkg.toml
Comment (palash25) [inline Gopkg.toml:19]: todo

File: services/management/checks.go
Comment (palash25) [inline services/management/checks.go:157]: for now I am returning an empty string in case the docstring is not present, let me know if we expect and error here

File: services/management/checks.go
Comment (AlekSi) [inline services/management/checks.go:142]: Yep, we should move it to the platform->saas repos

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: services/management/checks.go
Comment (AlekSi) [reply services/management/checks.go]: https://jira.percona.com/browse/SAAS-429

File: Gopkg.toml
Comment (AlekSi) [reply Gopkg.toml:20]: Merged, please update

Comment (palash25) [reply Gopkg.toml]: updated

**Review by artemgavrilov — APPROVED** [review]

File: services/management/checks.go
Comment (artemgavrilov) [inline services/management/checks.go:103]: I'm not sure, but maybe it makes sense to extract description form checke when we dowload it, not each time when `ListSecurityChecks` invoked. I supose that `getCheckDescription` is expensive function.

File: services/management/checks.go
Comment (AlekSi) [reply services/management/checks.go:157]: That's ok. We have it validated on SaaS side, so there we should handle it gracefully, and returning empty string is the best we can do

File: services/management/checks.go
Comment (AlekSi) [reply services/management/checks.go:103]: I think we should make description a top-level YAML field for exactly this reason: https://jira.percona.com/browse/SAAS-429
But that can be done later.

File: services/management/checks.go
Comment (AlekSi) [inline services/management/checks.go:144]: We should pass our predefined functions there (`GetFuncsForVersion(1)`)

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: services/management/checks.go
Comment (palash25) [reply services/management/checks.go:144]: blocked by https://github.com/percona-platform/platform/pull/160

Comment (AlekSi) [reply services/management/checks.go:144]: saas repo updated

**Comment by palash25** [comment]
I am guessing this is the reason for my build failure  https://github.com/percona/pmm/pull/600#issuecomment-726923216

**Comment by palash25** [comment]
the fix seems to be in this PR https://github.com/percona/pmm-managed/pull/510/files#diff-494e24211d6b35a6e1461aeb11f5218be2049c8c745f23ab541cbf9504c79102R304  but I am not sure when it will be merged so for now I am writing a dummy method just to satisfy the interface and make the build pass so that we can at least review this

**Comment by palash25** [comment]
tests fail because the PR is blocked by this https://github.com/percona-platform/checked/pull/106

**Comment by palash25** [comment]
tested it locally with a checks file and it works as expected

File: Gopkg.toml
Comment (palash25) [inline Gopkg.toml:19]: todo

File: services/management/grpc/actions_server.go
Comment (palash25) [inline services/management/grpc/actions_server.go:297]: just there to be able to build the code, revert it before merge

File: Gopkg.toml
Comment (AlekSi) [reply Gopkg.toml:19]: Merged, please update

**Review by AlekSi — APPROVED** [review]

**Review by AlekSi — CHANGES_REQUESTED** [review]
Tests are failing. At least several of them check local file, not SaaS.

**Comment by palash25** [comment]
fixed

**Review by AlekSi — APPROVED** [review]

**Pull request merged**
